src/proper.erl

```diff
@@ -1173,6 +1272,45 @@ inner_test(RawTest, Opts) ->
 	false -> ShortResult
     end.
 
+%% @private
+%% @doc Runs PropEr in parallel mode, through the use of workers to perform the tests.
+%% Under this mode, PropEr will detect if a property is pure or impure based on the passed options.
```

Comment:
"PropEr will detect if a property is pure or impure based on the passed options."  is not accurate.
"PropEr needs information whether a property is pure or impure, and this information is passed via an option." is a better sentence here, I think.  (Or something similar.)